src/main/java/shop/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,46 @@
+package shop.dao.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+import shop.dao.OrderDao;
+import shop.db.Storage;
+import shop.lib.Dao;
+import shop.model.Order;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order create(Order order) {
+        if (order == null) {
```

Comment:
@AndrewLinkii what do you mean here? maybe: 
```suggestion
        if (order != null) {
```

Reply:
please provide some examples in the main method